blog/2020-05-01-april-dev-update.md

```diff
@@ -0,0 +1,63 @@
+---
+title:  "April, 2020 - Dev Update"
+author: daniel
+author_url: https://github.com/da-kami
+author_image_url: https://avatars1.githubusercontent.com/u/5557790
+tags: [dev-update]
+---
+
+> It’s difficult to make predictions, especially about the future.
+>
+> Niels Bohr
+
+Are we at the verge of plummeting into the biggest recession since the big crash of 1929? 
+And if so, are cryptocurrencies the answer, a safe-haven to put your money in?
+
+While we cannot look into the future and don't dare to predict it, we are inclined to believe that it is *likely* to see a high demand for crypto in the future, and an even higher one in the next few months. 
+Time to get COMIT out there so you don't have to trust those central exchanges, but keep your private keys where they should be: to yourself.
+Verify, don't trust!
+
+Join us for a discussion around advances in crypto, trustless trading and - as usual - various tech related topics in [next week Thursday's COMIT Community ☕️](https://calendar.google.com/event?action=TEMPLATE&tmeid=NHEzb2tkNDJtcXFzdnNyNG9oM2UzNGhpc3MgdGVueC50ZWNoX202dWp2OXQ4cGluajAxYmI2NXV0aHRmcnM0QGc&tmsrc=tenx.tech_m6ujv9t8pinj01bb65uthtfrs4%40group.calendar.google.com) - 7pm Sydney time!
+
+<!--truncate-->
+
+It would be an under-statement to say that the COMIT team is quite tech-focused.
+We breath tech, usually focusing on discussions around protocol implementations rather than philosophising about the global financial system.
+However, with the recent turmoil caused by Covid-19, we had a noticeable trend of discussing topics outside the tech sphere.
+
+While some plans, like filling up your bathtub with crude oil and getting paid for it, were fun but not the most practical, other questions, like should I invest now? - and if yes into what? - were interesting to the team.
+And we don't seem to be the only ones.
+We are used to cryptocurrencies being a volatile market - but recently the traditional stock market went up and down even crazier.
+Making quick money might, however, be a shallow tactic, given that we might just be at the start of the biggest bear market to date.
+By looking at [statistic of previous historical crashes](https://todaytrader.com/blog/stock-market-crash-preparation/) it becomes clear that we might not have reached the bottom yet, and the time to break-even can be in the decades.
+We love statistics, but as the quote of this post says - predicting the future is a bet.
+It can be an educated bet, but it is still a bet.
+In the end it's up to each individual to come to a conclusion on how to handle this crisis.
```

Comment:
:thinking: To separate it from the medical crisis.
```suggestion
In the end it's up to each individual to come to a conclusion on how to handle this financial crisis.
```

Reply:
Actually, I would choose to omit this sentence, given that a lot of people won't have a choice on how to handle that one either.